codegens/curl/README.md

```diff
@@ -34,7 +35,8 @@ var request = new sdk.Request('www.google.com'),  //using postman sdk to create
         trimRequestBody: true,
         multiLine: true,
         followRedirect: true,
-        longFormat: true
+        longFormat: true,
+        quoteType: '\''
```

Comment:
the quoteType option here should be `singlequote` or `doublequote`. This can be translated to `'` or `"` inside the module.

Reply:
Fixed